File: Programming Basics/For Loop - Exercises/08.TennisRanklist.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        double tournaments = double.Parse(Console.ReadLine());
        double ranklistPoints = double.Parse(Console.ReadLine());
        double startPoints = ranklistPoints;
        double wonCount = 0;

        for (int i = 0; i < tournaments; i++)
        {
            string placement = Console.ReadLine();
            
            if (placement == "W")
            {
                ranklistPoints += 2000;
                wonCount++;
            }
            else if (placement == "F")
            {
                ranklistPoints += 1200;
            }
            else if (placement == "SF")
            {
                ranklistPoints += 720; 
            }
        }
        Console.WriteLine($"Final points: {ranklistPoints}");
        Console.WriteLine($"Average points: {Math.Floor((ranklistPoints - startPoints) / tournaments):f0}");
        Console.WriteLine($"{wonCount / tournaments * 100:f2}%");
    }
}

File: Programming Basics - Python/For Loop - Exercises/08.TennisRanklist.py
import math

tournaments = int(input())
initial_points = int(input())

points = 0

tournaments_won = 0

for i in range(0, tournaments):
    stage = input()

    if stage == "W":
        points += 2000
        tournaments_won += 1

    elif stage == "F":
        points += 1200

    elif stage == "SF":
        points += 720

print(f"Final points: {initial_points + points}")
print(f"Average points: {math.floor(points  / tournaments)}")
print(f"{(tournaments_won / tournaments) * 100:.2f}%")